config/settings.yml

```diff
@@ -12,6 +12,13 @@ financial:
     csv: true
     xml: true
 
+converters:
+  default:
+    product_to_journal_rows: Converters::ProductToJournalRows
+    order_detail_to_journal_rows: Converters::OrderDetailToJournalRows
+  split_accounts/split_account:
+    order_detail_to_journal_rows: SplitAccounts::Converters::OrderDetailToJournalRows
```

Comment:
I was torn.  Adding the split account settings here is much more clear, and it's non-obtrusive even though the engine is bleeding outside of it's boundaries a bit.  Moving the settings to the engine make me think I'll create more confusion than it's worth.  What do you think we should do?